monitor/services/src/main/java/org/datacleaner/monitor/server/controllers/ComponentControllerV1.java

```diff
@@ -443,6 +481,15 @@ static private String getURLForCreation(String tenant, ComponentDescriptor<?> de
         }
     }
 
+    private boolean isComponentEnabled(String componentName){
+        try {
+            canCall(componentName);
+        }catch (RuntimeException e){
+            return false;
+        }
+        return true;
+    }
```

Comment:
It probably doesn't matter in this case, but exceptions as part of normal flow is usual very slow (or, at least it used to be, not sure if they optimized it), and in general frowned upon. Isn't there a more proper way to interrogate the current enabled state?